### ISSUE TITLE
The error in the calculation of AUC

### ISSUE THREAD

**Issue by wt12318** [deepfindr/gnn-project#3]
Hi,

Thank you for this excellent GNN project. But In the `train.py` file, line 106 shows:

```
roc= roc_auc_score(y_pred,y_true)
```

According to [sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html), the first argument should be true label and second argument should be predicted score, so is it something wrong? The correct version could be:

```
roc=roc_auc_score(y_true,all_preds_raw)
```

**Comment by deepfindr** [comment]
Hi! Thanks for pointing out! You are absolutely right.

**Comment by deepfindr** [comment]
I've updated the code :)
